docs/module/ip-address-blocker-rules.md

```
@@ -15,6 +23,13 @@ PeerBanHelper 的重要模块之一，默认订阅来自 [PBH-BTN/BTN-Collected-
 
 ![rules-sub-logs](./assets/sub-rules-logs.png)
 
+## 触发条件
+
+- **管道**：种子处于[活跃传输状态](#生效范围)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)